### DISCUSSION TITLE
Build and View Timing of Tex Files

### DISCUSSION THREAD

**Discussion by The-Cyber-Thinkerer** [texstudio-org/texstudio#3327] [Q&A]
I am fairly new to this editor and would like some help. I have checked the mark which auto-recompiles document on change option on preferences (options). I am wondering if there is way to make the timing between my change and recompiling smaller (faster). In other other i want to make the process more instantaneous without any lag with each letter change. I know that this is not ideal specially for big tex files but i fairly work in small batches without any bibliography or complex content that might need many compilations. Thank you for your time!

**Comment by sunderme** [comment]
the delay is set in option/preview/auto update delay.
Default is 300ms.
